client/cmd/dexcctl/sample-dexcctl.conf

```diff
@@ -7,13 +27,30 @@
 ; proxyuser=
 ; proxypass=
 
-; Username and password to authenticate connections to a Dexc RPC server
+; Username and password to authenticate connections to a Dexc RPC server.
 ; rpcuser=
 ; rpcpass=
 
-; RPC server to connect to
+; RPC server to connect to.
 ; rpcaddr=localhost:5757
 
-; RPC server certificate chain file for validation
+; RPC server certificate chain file for validation.
 ; rpccert=~/.dexc/rpc.cert
 
+; ------------------------------------------------------------------------------
+; General settings
+; ------------------------------------------------------------------------------
+
+; Display version information and exit.
+; version=true
+
+; List all of the supported commands and exit.
+; listcommands=true
```

Comment:
Feels wrong to suggest these in the sample config like this.  I'm noting that dcrd also does not include `version` (https://github.com/decred/dcrd/blob/master/sampleconfig/sampleconfig.go).
However, I understand the desire to make the sample config as informative as possible, so I'd be good with just describing these switches near the top of the sample in more free-form introduction.